pkg/printers/internalversion/printers.go

```diff
@@ -410,6 +410,7 @@ func AddHandlers(h printers.PrintHandler) {
 	storageClassColumnDefinitions := []metav1beta1.TableColumnDefinition{
 		{Name: "Name", Type: "string", Format: "name", Description: metav1.ObjectMeta{}.SwaggerDoc()["name"]},
 		{Name: "Provisioner", Type: "string", Description: storagev1.StorageClass{}.SwaggerDoc()["provisioner"]},
+		{Name: "ReclaimPolicy", Type: "string", Description: storagev1.StorageClass{}.SwaggerDoc()["reclaimPolicy"]},
```

Comment:
I spoke with storage and they suggested adding also `AllowVolumeExpansion` and `VolumeBindingMode` wdyt?

Reply:
Thanks for view, I would  add it. 

I also think it is needed, `get result` of StorageClass is too simple  now, it just contains the name and provisioner.

Reply:
@soltysh I have updated the PR